package.json

```diff
@@ -60,8 +60,7 @@
     "process": "^0.11.10",
     "rimraf": "^3.0.2",
     "stylemark": "^3.1.7",
-    "vue-i18n-extract": "^2.0.4",
-    "vue3-datepicker": "^0.3.2"
```

Comment:
Is this related to this PR?

Reply:
Not really, it was unused since #95

I can rollback and do this in a new PR.

Reply:
No need I think, a message on the PR should be enough!

Reply:
Done !